.github/workflows/sync-fork.yml

```diff
@@ -0,0 +1,57 @@
+name: Sync Fork
+
+on:
+  schedule:
+    - cron: '0 0 */3 * *'  # runs at midnight every 3 days     
+
+jobs:
+  sync:
+    permissions: write-all
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: tgymnich/fork-sync@v1.8
+        with:
+          owner: SDSC-ORD
+          base: master
+          head: TopQuadrant:master
+          ignore_fail: true
+          retries: 1
+          retry_after: 15
+  fetch-release:
+    permissions: write-all
+    runs-on: ubuntu-latest
+
+    steps:
+     - name: Checkout repository
+       uses: actions/checkout@v3
+       with:
+          repository: 'TopQuadrant/shacl'
+          
+     - name: Get Version Tag
+       id: get_version_tag
+       run: |
+          git fetch --tags
+          git fetch --prune --unshallow || true
+          
+          LATEST_RELEASE=$(git describe --abbrev=0 --tags )
+          echo "latest-release=${LATEST_RELEASE}" >> $GITHUB_OUTPUT
+          echo "$LATEST_RELEASE"
+    outputs:
+      tag: "${{ steps.get_version_tag.outputs.latest-release }}"   
+  
+          
+  publish:
+    needs: [fetch-release, sync] 
+    runs-on: ubuntu-latest
+    
+    steps:
+     - name: Checkout repository
+       uses: actions/checkout@v3
+             
+     - name: automated_release_with_tag
+       uses: "marvinpinto/action-automatic-releases@latest"

```

Comment:
For the sake of consistency (and prevent any issue with broken/different "major" updates) it might be better to use `marvinpinto/action-automatic-releases@v1.2` here

Reply:
Ah, so not to grab the "latest" version but stick to a specific version (which in this case just happens to be the latest version). That's smart indeed. Will change it.